app/code/Magento/CatalogCustomerGraphQl/Model/Resolver/Product/Price/Tiers.php

```diff
@@ -97,7 +115,23 @@ public function getProductTierPrices($productId): ?array
         if (empty($this->products[$productId])) {
             return null;
         }
-        return $this->products[$productId]->getTierPrices();
+
```

Comment:
Is there a reason that the original Price::getTierPrices method does not for us here?

Reply:
this original Price::getTierPrices actually loads always in base currency not minding the current store selected currency.
Now, trying to see what can be done about it - maybe this method can be improved.